src/main/java/org/jitsi/jigasi/sip/SipInfoJsonProtocol.java

```diff
@@ -0,0 +1,101 @@
+package org.jitsi.jigasi.sip;
+
+import net.java.sip.communicator.service.protocol.CallPeer;
+import net.java.sip.communicator.service.protocol.OperationFailedException;
+import net.java.sip.communicator.service.protocol.OperationSetJitsiMeetTools;
+import net.java.sip.communicator.util.Logger;
+import org.jitsi.jigasi.JvbConference;
+import org.json.simple.JSONObject;
+
+import java.util.HashMap;
+
+/**
+ *
+ * Message JSON format
+ *
+ * Mandatory header :
+ *
+ * {
+ *      i: <Message Id>,
+ *      t: <Message Type>,
+ *      d: <Data>
+ * }
+ *
+ * General rule : Keys are represented using few letters.
+ *
+ * <Message Type> integer to identify message type. This integer should be documented.
+ *
+ * <Message Id> simple integer counter
+ *
+ * <Data> OPTIONAL data can be any type respecting the message format.
+ *
+ *
+ * This class is used to define sip info protocol.
+ */
+public class SipInfoJsonProtocol
```

Comment:
If you are encapsulating json communication in such messages for lobby, you should do the same and for the mutes, to keep consistency.

Reply:
Yes, agree, but thought this can be done later...